bib/internal/utils/fileutils.go

```diff
@@ -0,0 +1,14 @@
+package utils
```

Comment:
Not a fan of a `utils` package.  It's also generally frowned upon.
Also the commit message says "fix linter problems" but there's a lot more going on here than just linter fixes (a new package is introduced, for one).

Also, I think you meant that problems _arose_, not _arouse_ in the commit message :)